docs/pages/api-docs/datagrid.md

```diff
@@ -0,0 +1,109 @@
+# DataGrid API
+
+<p class="description">The API documentation of the DataGrid React component. Learn more about the props and the CSS customization points.</p>
+
+## Import
+
+```js
+import { DataGrid } from '@material-ui/data-grid';
+```
+
+## Props
```

Comment:
What do you think about adding `pagination`, `disableColumnResize`, `disableMultipleColumnsSorting`, `disableMultipleSelection` to the only value supported and mentioned that it can be changed on the XGrid? It would create noise but could also create awareness of the existing X features. So I'm not sure. Happy to keep it the way your proposed, could be simpler overall :)

Reply:
Several approaches. We could also put the icon on the prop for the enterprise. Or leave it this way as it should be generated and when it will be, I believe it will do 2 pages